content/spin/v2/language-support-overview.md

```diff
@@ -13,7 +13,7 @@ This page contains information about language support for Spin features:
 
 {{ startTab "Rust"}}
 
-[Visit the Spin SDK reference documentation.](https://fermyon.github.io/rust-docs/spin/main/spin_sdk/index.html)
+**[📄 Visit the Rust Spin SDK reference documentation](https://fermyon.github.io/rust-docs/spin/main/spin_sdk/index.html) to see specific modules, functions, variables and syntax relating to the following Rust features.**
```

Comment:
For me the document symbol seemed a bit blurry and didn't draw the eye.  Might this look better with something bolder like a BIG HONKIN' ARROW or something like that?